update docs (#1645)
pomerium/pomerium: docs/docs/installation.md

---

> title: Download

Author: Hopefully more clear so the sidebar workflow is:

1. What is pomerium
2. How do I get it

sidebarDepth: 2
description: This article describes various ways to install pomerium
---

# Releases

Pomerium is shipped in multiple formats and architectures to suit a variety of deployment patterns. There are two binaries:

- `pomerium` is the primary server component. It is a monolithic binary that can perform the function of any [services mode](/reference/#service-mode).

> - `pomerium-cli` (optional) is command-line interface tools that can be used to generate service accounts and act as an authentication helper to enable things like [kubtctl proxying](topics/kubernetes-integration.md).

Reviewer: `is` should be `are`? Or maybe `is a comand-line interface that can be used`

> - `pomerium-cli` (optional) is command-line interface tools that can be used to generate service accounts and act as an authentication helper to enable things like [kubtctl proxying](topics/kubernetes-integration.md).

Reviewer 2: ```suggestion
- `pomerium-cli` (optional) is a command-line client for working with Pomerium.  Functions include generating service accounts, and acting as an authentication helper for tools like [kubtctl](topics/kubernetes-integration.md).
```

[[toc]]

> ## Pomerium server

Reviewer 2: ```suggestion
## pomerium
```

Otherwise, should change `## pomerium-cli` back to something like `Pomerium Client`, which pairs nicely with `Pomerium Server`.  Either way, I think the two titles should share format.

- `:master` : which will pull an image in sync with git's [master](https://github.com/pomerium/pomerium/tree/master) branch.

  ```bash
  docker pull pomerium/pomerium:latest
  ```

### Helm

Pomerium maintains a [helm](https://helm.sh) chart for easy Kubernetes deployment with best practices <https://helm.pomerium.io/>

> ```shell

Reviewer 2: ```suggestion
```bash
```

### Source

::: tip

Officially supported build platforms are limited by [envoy proxy](https://www.envoyproxy.io/). If you have an enoy binary for your platform in your path at start time, `pomerium` should function correctly.

:::

> ```shell

Reviewer 2: ```suggestion
```bash
```

./bin/pomerium --version
```

> ## Pomerium-cli

Reviewer 2: Should lower case both `pomerium` and `pomerium-cli` or leave as-is.

make
./bin/pomerium-cli --help
```

## Release cycle

The current release cycle is aligned on a monthly basis. Pre-`1.0.0` we target a `MINOR` release on or around the **first day of each month**. We try to hit the targets as closely as possible, while still delivering a quality release.

Pomerium uses [Semantic Versioning](https://semver.org/). In practice this means for a given version number **vMAJOR**.**MINOR**.**PATCH** (e.g. `v0.1.0`):

> - **MAJOR** indicates a incompatible API changes,

Reviewer: `a` should be `an` and `changes` should be `change`, or `a` should be removed entirely
